config/app-custom.yaml.template

```diff
@@ -0,0 +1,4 @@
+# Place any overrides of the default values, from app-default.yaml, into this file.
+app:
+  logging:
+    sql_log_level: "DEBUG"  # example override: set sql logging to DEBUG
```

Comment:
What happens to list values? Like if we override the activity_types, do we have to specify all the supported activity types?

Reply:
Yes, have to specify all of them.